README.md

```diff
@@ -210,6 +214,22 @@ All of Facebook SDK for Swift development happens on GitHub.
 Contributions make for good karma and we welcome new contributors with tremendous joy.
 We encourage you also to read our [contributing guidelines](https://github.com/facebook/facebook-sdk-swift/blob/master/CONTRIBUTING.md) before submitting a Pull Request.
 
+### Building the Facebook Swift SDK
+
+Build the SDK to make changes and contribute. Follow these steps to get started.
+
+1. First clone the sdk into an empty directory for the Facebook Swift SDK.
```

Comment:
nit: these instructions aren't helpful, since they just mirror the bash invocation below. I'd say remove them and instead add instructions on how to build the SDK (I.e. which target to build in the xcworkspace) and how to build any demo apps if those are included